edit-post/components/sidebar/style.scss

```diff
@@ -28,13 +28,15 @@
 		border-right: none;
 		overflow: auto;
 		-webkit-overflow-scrolling: touch;
-		height: 100%;
+		height: auto;
+		max-height: calc( 100vh - #{ $admin-bar-height-big + $panel-header-height } );
 		margin-top: -1px;
 		margin-bottom: -1px;
 
 		@include break-small() {
 			overflow: inherit;
 			height: auto;
+			max-height: unset;
```

Comment:
The `unset` keyword is not supported in IE11.

https://developer.mozilla.org/en-US/docs/Web/CSS/unset#Browser_compatibility

You want `max-height: none;`

https://developer.mozilla.org/en-US/docs/Web/CSS/max-height

Reply:
Thank you for the links and for catching this issue it was updated.